### DISCUSSION TITLE
Human Motion Recognition Using IMUs (Project 232)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#66] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #232](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Human%20Motion%20Recognition%20Using%20IMUs)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by rabouhia** [comment]
Hi, I am a student at Mustapha Stambouli and interested in working and collaborating on this project

**Comment by Vikas-ABD** [comment]
Hi, I am a student at  and interested in working and collaborating on this project

**Comment by Geonhui-Kang** [comment]
Hi, I am a researcher at Wonkwang University and interested in working and collaborating on this project.

**Reply by Vikas-ABD** [reply]
I love to join

**Reply by Geonhui-Kang** [reply]
> I love to join

Are you working on this project since 02/28/2023? I have eight IMU sensors in our laboratory.

**Comment by liumengyuan1997** [comment]
Hi all, I'm a student who is at Northeastern University. I would love to contribute to this project!!

**Comment by amaresh-1404** [comment]
My name is Amaresh H, a student from RV College of Engineering, Bangalore, and I would love the opportunity to contribute to this project!!
